src/editor/initialization/initialize-dom.js

```diff
@@ -1,48 +1,7 @@
-/* global IODIDE_EVAL_FRAME_ORIGIN */
-
 /* the code in this file initialize a few DOM elements for the react
-elements to mount. There are a few legacy versions of the DOM template
-we want to support
-
-v1: has a `#page` element that the whole app mounts,
-and no iframe#eval-frame yet
-
-v2: has an iframe inside of `#panes-container`
-
-v3: iframe#eval-frame is directly in <body>, and no other divs for react to mount
-
-as of v3: need to set up the DOM to have
-<body>
-  <div#notebook-header>
-  <div#editor-react-root>
-  <iframe#eval-frame>
-</body>
+elements to mount
 */
 
-const pageElt = document.getElementById("page");
-const panesContainerElt = document.getElementById("panes-container");
-let iframeElt = document.getElementById("eval-frame");
-// the following is provided for backward compatibility with old html
-// bundles that have a 'page' element and no 'eval-frame'
-if (iframeElt === null && pageElt !== null) {
-  // insert an iframe
-  iframeElt = document.createElement("iframe");
-  iframeElt.id = "eval-frame";
-  iframeElt.src = `${IODIDE_EVAL_FRAME_ORIGIN}/iodide.eval-frame.${process.env.IODIDE_VERSION}.html`;
-  iframeElt.setAttribute("sandbox", "allow-scripts allow-same-origin");
-  iframeElt.setAttribute("allowfullscreen", "true");
-  iframeElt.setAttribute("allowvr", "yes");
-  document.body.appendChild(iframeElt);
-}
-if (iframeElt !== null && panesContainerElt !== null) {
-  // in case of html template v2, nothing is required,
-  // but note that this will leave a legacy div#panes-container
-  // wrapping iframe#eval-frame
-}
-if (iframeElt !== null && panesContainerElt === null) {
-  // in case of html template v3, nothing is required
-}
-
```

Comment:
if we're cutting all the stuff above in this file to support legacy docs (which is totally the right thing to do at this point), we should also now put the two divs specified below directly into the body of the html templates we serve, so that we can eliminate this file entirely